src/server/api/controllers/users.controller.js

```diff
@@ -0,0 +1,33 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getUsers = async () => {
+  return knex('users').select(
+    'users.id',
+    'users.full_name',
+    'users.firebase_token',
+  );
+};
+
+const getUserById = async (id) => {
+  if (!id) {
+    throw new HttpError('Id should be a number', 400);
+  }
+
+  try {
+    const users = await knex('users')
+      .select('users.id as id', 'full_name', 'firebase_token')
+      .where({ id });
+    if (users.length === 0) {
+      throw new Error(`incorrect entry with the id of ${id}`, 404);
+    }
+    return users;
```

Comment:
if you only return one user it is should be named `user` and not `users` as that might be confusing :)

Reply:
and I would expect to receive an object of that user and not an array